cmd/entrypoints/serve.go

```diff
@@ -54,6 +56,11 @@ func serveInsecure(ctx context.Context, cfg *config.Config) error {
 func newGRPCServer(_ context.Context, cfg *config.Config) *grpc.Server {
 	grpcServer := grpc.NewServer()
 	datacatalog.RegisterDataCatalogServer(grpcServer, datacatalogservice.NewDataCatalogService())
+
+	healthServer := health.NewServer()
+	healthServer.SetServingStatus("", grpc_health_v1.HealthCheckResponse_SERVING)
+	grpc_health_v1.RegisterHealthServer(grpcServer, healthServer)
```

Comment:
👍 nice
nit: we can rename the http health check method below to `serveHTTPHealthcheck`, this is for integration into proxies like Envoy's health check

Reply:
PTAL again and help merge it. Thanks.